05/app.js

```diff
@@ -1 +1,14 @@
-const numbers = [1, 2, 3, 4, 5, 6, 7];
\ No newline at end of file
+const numbers = [1, 2, 3, 4, 5, 6, 7];
+
+const evenNumbers = numbers.filter(function (num) {
+    if (num % 2 === 0) {
+        return num;
+    }
```

Comment:
Tutaj chodzi, aby zwracać `true` lub `false` - bezpiecznej będzie zmienić bo jeśli byłby 0 to już mógłby być problem.

Można to nawet zapisać krócej: `return num % 2 === 0` => będzie zwrócony wynik z porównania